models/wholeBrainSeg_Large_UNEST_segmentation/docs/README.md

```diff
@@ -0,0 +1,143 @@
+# Description
+Detailed whole brain segmentation is an essential quantitative technique in medical image analysis, which provides a non-invasive way of measuring brain regions from a clinical acquired structural magnetic resonance imaging (MRI).
+We provide the pre-trained model for inferencing whole brain segmentation with 133 structures.
+
+A tutorial and release of model for whole brain segmentation using the 3D transformer-based segmentation model UNEST.
+
+Authors:
+Xin Yu (xin.yu@vanderbilt.edu) (Primary)
+
+Yinchi Zhou (yinchi.zhou@vanderbilt.edu) | Yucheng Tang (yuchengt@nvidia.com)
+
+<p align="center">
+-------------------------------------------------------------------------------------
+</p>
+
+![](./demo.png) <br>
+<p align="center">
+Fig.1 - The demonstration of T1w MRI images registered in MNI space and the whole brain segmentation labels with 133 classes</p>
+
+
+
+# Model Overview
+A pre-trained larger UNEST base model [1] for volumetric (3D) whole brain segmentation with T1w MR images.
+To leverage information across embedded sequences, ”shifted window” transformers
+are proposed for dense predictions and modeling multi-scale features. However, these
+attempts that aim to complicate the self-attention range often yield high computation
+complexity and data inefficiency. Inspired by the aggregation function in the nested
+ViT, we propose a new design of a 3D U-shape medical segmentation model with
+Nested Transformers (UNesT) hierarchically with the 3D block aggregation function,
+that learn locality behaviors for small structures or small dataset. This design retains
+the original global self-attention mechanism and achieves information communication
+across patches by stacking transformer encoders hierarchically.
+
+![](./unest.png) <br>
+<p align="center">
+Fig.2 - The network architecture of UNEST Base model
+</p>
+
+
+## Data
+The training data is from the Vanderbilt University and Vanderbilt University Medical Center with public released OASIS and CANDI datsets.
+Training and testing data are MRI T1-weighted (T1w) 3D volumes coming from 3 different sites. There are a total of 133 classes in the whole brain segmentation task.
+Among 50 T1w MRI scans from Open Access Series on Imaging Studies (OASIS) (Marcus et al., 2007) dataset, 45 scans are used for training and the other 5 for validation.
+ The testing cohort contains Colin27 T1w scan (Aubert-Broche et al., 2006) and 13 T1w MRI scans from the Child and Adolescent Neuro Development Initiative (CANDI)
+ (Kennedy et al., 2012). All data are registered to the MNI space using the MNI305 (Evans et al., 1993) template and preprocessed follow the method in (Huo et al., 2019). Input images are randomly cropped to the size of 96 × 96 × 96.
+
+### Important
+
+```diff
++ All the brain MRI images for training are registered to Affine registration from the target image to the MNI305 template using NiftyReg.
++ The data should be in the MNI305 space before inference.
```

Comment:
perhaps briefly mention the commands to achieve the registration?

Reply:
Yep, added a sample section for registration information. Thanks.